pkg/reconciler/buildrun/resources/build_test.go

```diff
@@ -130,22 +130,22 @@ var _ = Describe("Build Resource", func() {
 
 	Context("Operating on embedded Build(Spec) resources", func() {
 		client = &fakes.FakeClient{}
-		buildRun := &build.BuildRun{
+		buildRun := &buildapi.BuildRun{
 			ObjectMeta: metav1.ObjectMeta{
 				Name:      "foo",
 				Namespace: "bar",
 			},
-			Spec: build.BuildRunSpec{
-				Build: build.ReferencedBuild{
-					Spec: &build.BuildSpec{
+			Spec: buildapi.BuildRunSpec{
+				Build: buildapi.ReferencedBuild{
+					Spec: &buildapi.BuildSpec{
 						Env: []v1.EnvVar{{Name: "foo", Value: "bar"}},
 					},
 				},
 			},
 		}
 
 		It("should be able to retrieve an embedded build object if it exists", func() {
-			build := &build.Build{}
+			build := &buildapi.Build{}
```

Comment:
At this point, I feel like the rename adds a lot of noise to the PR. Would it be possible that you defer the import name change here for another PR, so that we can keep this one very focused?

Reply:
I say that, because there are some other improvements in-between these changes, like using a constant instead of the hardcoded string, but it is hard to see in this sea of import name changes.

Reply:
I reverted the commit where I made the import name changes. I will open a separate PR for it.